pizzabot.js

```diff
@@ -0,0 +1,83 @@
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
+
+document.getElementById(
+  "botBubble"
+).innerHTML = `Hey! Happy to get your order started. On our menu we have ${vegetarian}, ${hawaiian}, and ${pepperoni}.`;
```

Comment:
Interesting line break, I would rather suggest :

```
document.getElementById("botBubble").innerHTML =
    `Hey! Happy to get your order started. On our menu we have ${vegetarian}, ${hawaiian}, and ${pepperoni}.`;
```

Reply:
updated this.